File: 30 Days of code/src/fr/tenji/java/Days9.java
package fr.tenji.java;

import java.util.Scanner;

/**
 * Created by nekran on 12/10/16.
 */
public class Days9 {
    private static int factorial(int n) {
        if (n <= 1) {
            return 1;
        } else {
            return n * factorial(n - 1);
        }
    }
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        in.close();
        System.out.println(Days9.factorial(n));
    }
}

File: 30 Days of code/src/fr/tenji/js/days9.js
/**
 * Created by nekran on 12/10/16.
 */
function processData(input) {
    //Enter your code here
    console.log(factorial(input));
}
function factorial (n) {
    if (n <= 1) {
        return n;
    } else {
        return n * factorial(n - 1);
    }
}
process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
    process.stdin.end();
});

process.stdin.on("close", function () {
    processData(_input);
});
